ons_alpha/release_calendar/models.py

```diff
@@ -17,27 +16,26 @@
 
 
 class ReleaseStatus(models.TextChoices):
-    UPCOMING = "UPCOMING", "Upcoming"
-    PUBLISHED = "PUBLISHED", "Published"
-    CANCELLED = "CANCELLED", "Cancelled"
+    PROVISIONAL = "PROVISIONAL", _("Provisional")
+    CONFIRMED = "CONFIRMED", _("Confirmed")
+    CANCELLED = "CANCELLED", _("Cancelled")
+    PUBLISHED = "PUBLISHED", _("Published")
```

Comment:
note: re-added the PUBLISHED status as we vary the content, as well as we mark release pages as "published" when publishing a bundle